### ISSUE TITLE
Venues should receive an email notification once they've been successfully listed

### ISSUE THREAD

**Issue by ctrembath** [club-soda/club-soda-guide#88]
As a venue,
I should receive an email notification informing my that my venue listing was successfully submitted,
so that I know my venue will now be listed on the Club Soda Guide

**Comment by iteles** [comment]
@jessyclubsoda You mentioned this issue in an email to me today.
Seeing as a venue now receives a password setting email when they sign up and that this happens as part of the process when they first set up a new venue, do you want _another_ email to be sent to them? 

Should we consider updating the wording in the password setting email to include words to the effect of 'Your venue has been set up successfully! Now all you need to do is set up your password so that ...... ' rather than having a separate email sent to venue managers for this?

**Comment by jessyclubsoda** [comment]
@iteles Yeah I think just having a sentence like that would be perfect - we don't want to overwhelm venues with emails anyway so I think a confirmatory sentence will be fine

**Comment by nelsonic** [comment]
@RobStallion please update the copy in the email when you can. thanks! ✨

**Comment by RobStallion** [comment]
@jessyclubsoda is the following okay?

Your venue has been set up successfully!
Now all you need to do is use the link below to verify your account and set up your password.
www.link_to_club_soda.com (this would be a real link in the email of course)

Also updated the email subject from "Club Soda account verification" to "Venue Created". Let me know what you prefer 👍

**Comment by jessyclubsoda** [comment]
@RobStallion Sounds great! and yes 'Venue Created' better than 'CS account ver...' I think

**Comment by RobStallion** [comment]
That was really quick. Thanks. I'll add that in now 👍
